### PR TITLE
Datadog log

### PR DESCRIPTION
**Pull request by quangnhut123**

### CODE REVIEW

**Comment by ronaldtse** [comment]
@quangnhut123 thank you for the contribution!

@phuonghuynh and @kwkwan can you check if we can merge this? Thanks!

File: ecs_datadog.tf [hunk 1]
```diff
@@ -9,14 +9,43 @@ resource "aws_ecs_task_definition" "datadog" {
...
+    ],
+    "environment": [
+     {
       "name" : "DD_API_KEY",
       "value" : "${var.datadog-api-key}"
-    }],
-    "command": [
-      "bash",
-      "-c",
-      "${var.datadog-extra-config}"
```
Comment (phuonghuynh) [inline ecs_datadog.tf:19]: @quangnhut123 is there any reason to remove this command?

File: ecs_datadog.tf [hunk 2]
```diff
@@ -47,7 +76,7 @@ EOF
 
   volume {
     name      = "cgroup"
-    host_path = "/cgroup/"
+    host_path = "/sys/fs/cgroup/"
```
Comment (phuonghuynh) [inline ecs_datadog.tf:79]: I not sure this location, in some other system, the path might be "/cgroup" ; [link](https://www.datadoghq.com/blog/how-to-collect-docker-metrics/)

@quangnhut123 i think we should introduce new variable for this location and update README for this?

File: variables.tf [hunk 3]
```diff
@@ -1,5 +1,5 @@
 variable "env" {}
 variable "identifier" {}
 variable "datadog-api-key" {}
-variable "datadog-extra-config" {}
 variable "ecs-cluster-id" {}
+variable "datadog_aws_integration_external_id" {}
```
Comment (phuonghuynh) [inline variables.tf:5]: We also need to update README for this variable as well

File: aws_datadog_iam.tf [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+        "${var.datadog_aws_integration_external_id}"
+      ]
+    }
+  }
+}
+
+data "aws_iam_policy_document" "datadog_aws_integration" {
+  statement {
+    actions = [
+        "apigateway:GET",
```
Comment (phuonghuynh) [inline aws_datadog_iam.tf:23]: @quangnhut123 is it better to refactor this list as a variable to let ppl easy config which ones they would like to monitor ?

**Comment by phuonghuynh** [comment]
Thanks @quangnhut123. I have some comments for you.

**Pull request closed without merging**